test/LaplaceApproximationModule.jl

```diff
@@ -264,4 +264,13 @@
         res = res_array[end]
         @test res.q isa MvNormal
     end
+
+    @testset "GitHub issue #109" begin
+        build_latent_gp() = LatentGP(GP(SEKernel()), BernoulliLikelihood(), 1e-8)
+
+        x = ColVecs(randn(2, 5))
+        _, y = rand(build_latent_gp()(x))
+
+        build_laplace_objective(build_latent_gp, x, y)()
```

Comment:
Maybe test that the produced objective function works as well?

Reply:
it is actually called:) but I'll make that clearer